apps/mobile/src/components/DrawerNavigation/Sidebar.tsx

```diff
@@ -0,0 +1,53 @@
+import type { FC } from 'react';
+import { View } from 'react-native';
+import type { DrawerContentComponentProps } from '@react-navigation/drawer';
+import DashboardNavigator from '@walless/app/features/DashboardLayout/Navigator';
```

Comment:
don't use this, export `DashboardNavigator` in `@walless/app`, and then:
import { DashboardNavigator } from '@walless/app'